packages/composite-checkout/src/components/billing-fields.js

```diff
@@ -0,0 +1,525 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { useLocalize } from '../lib/localize';
+import { useCheckoutLineItems } from '../index';
+import GridRow from './grid-row';
+import Field from './field';
+
+export default function BillingFields( {
+	summary,
+	setPaymentData,
```

Comment:
Sorry to do this to you, but #37134 changes this so that instead of `setPaymentData` this will get a `dispatch` prop, which you'll call like this:

```js
		dispatch( {
			type: 'PAYMENT_DATA_UPDATE',
			payload: { billingName: value },
		} );
```

However, I _think_ you can add this helper here for now and you shouldn't have to modify anything else.

```js
	const setPaymentData = payload =>
		dispatch( {
			type: 'PAYMENT_DATA_UPDATE',
			payload,
		} );
```

Reply:
No problem. Thanks for updating and merging it.